data/wanderers start.txt

```diff
@@ -242,33 +242,10 @@ mission "Visit Wanderers Again"
 	
 	on accept
 		set "language: Wanderer"
+		set "license: Wanderer Outfits"
```

Comment:
This should probably be added to the new "Wanderer Hail Patch" mission, as well, so old saved games will have access to Wanderer tech.